apps/src/music/views/MusicView.jsx

```diff
@@ -61,6 +61,8 @@ class UnconnectedMusicView extends React.Component {
     // used to differentiate tracks created on the same trigger
     this.triggerCount = 0;
 
+    this.isExecutingPlay = false;
```

Comment:
Could we just use `this.state.play` in place of this?

Reply:
fwiw, we need to set it before `state.isPlaying` is set to `true`...

Reply:
Revisiting this, and it turns out that since we will render the timeline elements (which use this) differently depending when playing or not, your original suggestion was right!